.github/workflows/build-and-test.yml

```diff
@@ -784,6 +775,86 @@ jobs:
         if: ${{ failure() && runner.debug }}
         uses: canonical/action-tmate@main
 
+  juju-remote-relation-test:
+    needs:
+      - lint
+      - unit-test
+      - build
+    name: Remote Integration test
+    runs-on: self-hosted-linux-amd64-noble-xlarge
+    steps:
+      - name: Download charm
+        uses: actions/download-artifact@v4
+        with:
+          name: charms
+          path: ~/artifacts/
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup LXD
+        uses: canonical/setup-lxd@v0.1.1
+        with:
+          # pin lxd to LTS release.
+          channel: 5.21/stable
+
+      - name: Install dependencies
+        run: ./tests/scripts/ci_helpers.sh install_deps
+
+      - name: Install MicroCeph charm
+        run: |
+          set -eux
+          date
+          # Deploy 2 microceph clusters named primary and secondary
+          juju deploy ~/artifacts/microceph.charm primary --storage osd-standalone='2G,3' --constraints="virt-type=virtual-machine root-disk=50G mem=8G"
+          juju deploy ~/artifacts/microceph.charm secondary --storage osd-standalone='2G,3' --constraints="virt-type=virtual-machine root-disk=50G mem=8G"
```

Comment:
noble xlarge runners have a rootfs of 100G, 2x50 for VMs will not actually be usable. Do we need that much? In my xp 8G rootfs + space for loop OSDs should be enough

Reply:
let me try with a smaller size, fme, these are thin provisioned so shouldn't be a problem for short lived clusters.